x/clerk/module.go

```diff
@@ -48,10 +47,6 @@ func (AppModuleBasic) Name() string {
 	return types.ModuleName
 }
 
-func (AppModuleBasic) RegisterCodec(cdc *codec.LegacyAmino) {
```

Comment:
why did you remove this?

Reply:
in AppModuleBasic interface there is no  `RegisterCodec` func